scripts/assets/test_file_exemptions.textproto

```diff
@@ -815,4 +815,4 @@ exempted_file_path: "utility/src/main/java/org/oppia/android/util/logging/perfor
 exempted_file_path: "utility/src/main/java/org/oppia/android/util/logging/performancemetrics/PerformanceMetricsEventLogger.kt"
 exempted_file_path: "utility/src/main/java/org/oppia/android/util/logging/performancemetrics/PerformanceMetricsLogScheduler.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/oppialogger/logscheduler/MetricLogSchedulingWorkerFactory.kt"
-exempted_file_path: "domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/PerformanceMetricsLoggerModule.kt"
+exempted_file_path: "domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/ActivityLifecycleObserverModule.kt"
```

Comment:
We should be testing the module, but I think I left this as a comment in the previous PR.

Reply:
Will fix it in the previous PR first and it'll reflect here.

Reply:
Reminder to remove this once it's addressed and merged into this PR.

Reply:
done.